client/src/components/devices/ChooseAuthentication.vue

```diff
@@ -1,77 +1,175 @@
+      </div>
+
+      <div v-if="authentication === DeviceSaveStrategyTag.Password">
+        <div class="method-chosen">
+          <ion-text class="method-chosen__title subtitles-sm">{{ $msTranslate('Authentication.methodChosen') }}</ion-text>
+          <authentication-card
+            :image-src="ellipsisGradient"
+            image-alt="ellipsis icon"
+            method-name="Authentication.method.password"
+            :state="State.Update"
+            @update-clicked="changeAuthenticationMethod"
+          />
+        </div>
+        <ms-choose-password-input
+          ref="choosePassword"
+          class="choose-password"
+          @on-enter-keyup="$emit('enterPressed')"
+        />
+      </div>
+
+      <!-- PKI Authentication -->
+      <!-- Replace v-if with authentication === DeviceSaveStrategyTag.Pki when available -->
+      <div v-if="false">
+        <div class="method-chosen">
+          <ion-text class="method-chosen__title subtitles-sm">{{ $msTranslate('Authentication.methodChosen') }}</ion-text>
+          <authentication-card
+            :image-src="idCardGradient"
+            image-alt="id card icon"
+            method-name="Authentication.method.smartcard"
+            :state="State.Update"
+            @update-clicked="changeAuthenticationMethod"
+          />
+        </div>
+      </div>
+
+      <!-- SSO Authentication -->
+      <!-- Replace v-if with authentication === DeviceSaveStrategyTag.Sso when available -->
+      <div v-if="false">
+        <div class="method-chosen">
+          <ion-text class="method-chosen__title subtitles-sm">{{ $msTranslate('Authentication.methodChosen') }}</ion-text>
+          <authentication-card
+            :image-src="personCircleGradient"
+            image-alt="person circle icon"
+            method-name="Authentication.method.ssoProConnect"
+            :state="State.Update"
+            @update-clicked="changeAuthenticationMethod()"
+          />
+        </div>
+      </div>
     </div>
-  </ion-list>
+  </div>
 </template>
 
 <script setup lang="ts">
-import { MsChoosePasswordInput } from 'megashark-lib';
+import { MsChoosePasswordInput, Translatable } from 'megashark-lib';
 import KeyringInformation from '@/components/devices/KeyringInformation.vue';
 import { DeviceSaveStrategy, DeviceSaveStrategyTag, SaveStrategy, isDesktop, isKeyringAvailable, isWeb } from '@/parsec';
 import { IonList, IonRadio, IonRadioGroup, IonText } from '@ionic/vue';
 import { onMounted, ref, useTemplateRef } from 'vue';
-
+import authenticationCard from '@/components/profile/AuthenticationCard.vue';
+import keypadGradient from '@/assets/images/keypad-gradient.svg';
+import idCardGradient from '@/assets/images/id-card-gradient.svg';
+import ellipsisGradient from '@/assets/images/ellipsis-gradient.svg';
+import personCircleGradient from '@/assets/images/person-circle-gradient.svg';
+import { State } from '@/components/profile/AuthenticationCard.vue';
+
+const methodSelected = ref(false);
```

Comment:
Just use the `authentication` variable and initialize it at `undefined`: if `authentication` is undefined, no method has been selected. You don't need another variable.